modules/mirror-registry-localhost.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-localhost{context}"]
+= Mirroring on a local host with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a local host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://console.redhat.com/openshift/downloads[*Downloads*] page of {product-title}.
+
+. Install Red Hat Quay on your local host with your current user account by using the `mirror-registry` tool:
++
+[source,terminal]
+----
+$ sudo ./mirror-registry install \
+  --quayHostname <host_example_com> \
+  --quayRoot <example_directory_name>
+----
+
+. The Red Hat Quay console is accessible at `\https://<host.example.com>:8443` after installation. Use the user name and password generated during installation to log into the registry by running the following command:
++
+[source,terminal]
+----
+$ podman login -u init \
+  -p <password> \
+  --tls-verify=false <1>
+----
+<1> You can avoid running `--tls-verify=false` by configuring your system to trust the generated rootCA certificates. See "Using SSL to protect connections to Red Hat Quay" and "Configuring the system to trust the certificate authority" for more information.
```

Comment:
Where are these mentioned doc chapters?

Reply:
In the 'Additional Resources' section below the Flags section. This is how OCP docs have to add additional resources; x-refs can't be added in modules because they can potentially break the build.

Reply:
It's also been deemed unnecessary to say something like "See xyz in the Additional Resources section of this document."